packages/examples/cvat/exchange-oracle/src/core/config.py

```diff
@@ -207,3 +254,27 @@ class Config:
     storage_config = StorageConfig
     features = FeaturesConfig
     core_config = CoreConfig
+    encryption_config = EncryptionConfig
+
+    @classmethod
+    def validate(cls) -> None:
+        for attr_or_method in cls.__dict__:
+            attr_or_method = getattr(cls, attr_or_method)
+            if inspect.isclass(attr_or_method) and issubclass(attr_or_method, _BaseConfig):
+                attr_or_method.validate()
+
+    @classmethod
+    def get_network_configs(
+        cls, only_configured: Optional[bool] = None
```

Comment:
I suggest that this parameter is set to true by default and accepts only true and false. False returns all, and true returns only configured. Currently, there is only 1 use case for the function, which needs this parameter set to true. I don't see any use cases for false and none distinction here at the moment.